### ISSUE TITLE
CLAIMED-SCORE - flag to disable exception

### ISSUE THREAD

**Issue by darvark** [thxo/cabrillo#14]
HI,
Since no all contest logging software properly generate cabrillo files, it's often situation when logs sent to scoring doesn't have all header fields fullfiled. Can you please add some flag to ommit errors like "cabrillo.errors.InvalidLogException: Improperly formatted claimed score "
In my tests for over 770 log files over 34 has CLAIMED-SCORE empty. (Logs taken from publiclogs for CQ WPX SSB 2024).

Thanks

**Comment by darvark** [comment]
Maybe such workaround is acceptable


![obraz](https://github.com/user-attachments/assets/dd53ead9-e193-4537-9465-987dd52d9cbf)

Additionality parser crashes when there is an empty line in log, Most modern software doesn't put empty lines to log file, but some do. This should not crash parser.  I'll create pull request with some changes,

**Comment by thxo** [comment]
Thank you for reporting this. Do you happen to have the link to these public logs? Would be good to add them as test cases.

**Comment by thxo** [comment]
I've cut a new release v0.2.1 that addresses this.

**Comment by darvark** [comment]
I got only part of public logs from CQWPX SSB 2024.
Empty claimed score I've found in:
om1alt, sv2hjq, ui4p, sv6hef, sp9kju, ry3d, ea2ejo, wb2zab, f4jnu, m0dia, 4k6fo. And probably in many many more.
Empty lines are in:
hb9cve, rc0cd, oe2xal, ji2kxk (at least from those I have).

Generally it's a nice play to fetch public log for on one stations, and then download all logs for contacts. It can be done with simply wget command
